price/domain/price_test.go

```diff
@@ -249,3 +249,15 @@ func TestCharges_Add(t *testing.T) {
 		Type:  "main",
 	}, charge)
 }
+
+func TestPrice_SplitInPayables_NegativeValue(t *testing.T) {
```

Comment:
We could add this edge case to the TestPrice_SplitInPayables so that all test data related to the function are in one test func.